servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -266,6 +266,9 @@ public HttpExecutionStrategy executionStrategy() {
             connectionFilterFactory = appendConnectionFilter(connectionFilterFactory,
                     HttpMessageDiscardWatchdogClientFilter.INSTANCE);
 
+            // TODO: maybe response classification is a more broad concept and we really need to be able to do
+            //  apply it to the ConnectionFactory.
```

Comment:
what do you mean by this comment?

Reply:
We don't have the all the features right now nor the idea of letting users define the classification, but in an ideal world multiple concepts want to be able to classify a response including the LB, the retry filter, observability tools, etc.